index.html

```diff
@@ -152,6 +152,14 @@ <h3 >
                 </button>
                 <p>Level Passed!</p>
                 <p>Respect++</p>
+                <div class="greeting-buttons">
+                    <button class="visual-button" onclick="playAgain()" type="button" title="Play again">
+                        <i class="fas fa-redo-alt fa-2x"></i>
+                    </button>
+                    <button class="visual-button" onclick="nextLevel()" type="button" title="Next level">
+                        <i class="fas fa-play fa-2x" aria-hidden="true"></i>
```

Comment:
use this instead `<i class="fas fa-angle-double-right fa-2x" aria-hidden="true"></i>`